doc/aspice_swe3_4/example/iceoryx_component/include/example_module/example_derived_class.hpp

```diff
@@ -61,6 +61,10 @@ class ExampleDerivedClass : public ExampleBaseClass<T>
     /// @param[in] b Description of input parameter b
     ExampleDerivedClass(const uint64_t a, const uint64_t b) noexcept;
 
+    /// @copydoc ExampleBaseClass<T>::doSomething
```

Comment:
Looks good!
Would may extend it to a case where argument types are used like described [here](https://www.doxygen.nl/manual/commands.html#cmdcopydoc)?
This make it more usable and avoid the trap that when specifying the types no spaces are allowed.

Reply:
@dkroenke shall I also encourage the use of `@copybrief` and `@copydetail` in case the base documentation should be used only partly?

Reply:
I would limit the copies only to the necessary things like virtual methods, otherwise people could begin to copy wildly documentation texts from somewhere else which makes it harder to follow when reading.